ui/pages/onboarding-flow/welcome/index.scss

```diff
@@ -51,4 +51,9 @@
       margin-bottom: 24px;
     }
   }
+
+  &__terms-of-use{
```

Comment:
If you use `<Box as="a">` you can put these properties on the Box and not in CSS.